src/stemmablenote.ts

```diff
@@ -22,7 +22,7 @@ export abstract class StemmableNote extends Note {
   protected flag?: Glyph;
   protected stem_extension_override?: number;
 
-  constructor(note_struct: NoteStruct) {
+  constructor(note_struct: Partial<NoteStruct>) {
```

Comment:
note_struct => noteStruct :-)

I figure we might as well rename it now while we are working on this PR. (I see stavenote.ts uses noteStruct).